docs/syntax.rst

```diff
@@ -97,15 +102,20 @@ is what a few other languages allow as well.
   # stenotype
   foo: "?int"
 
-----
+Containers
+''''''''''
+
+All containers are specified by their literal notation.
 
 Tuple_:
-Fixed size container where for every element present in it the correct type
-should be supplied. Tuples may also be defined with flexible length, which is
-useful for variadic functions. All containers should be specified by using
-their literal python notation equivalent.
+Fixed size ``()`` literal with element types specified for each position.
+Tuples may also be defined with flexible length using ``...`` to arbitrarily repeat the preceding type.
 
 .. code-block:: python
+  
+  # value
```

Comment:
with `value` blocks as well these sections start looking a little busy. I was thinking of including [sphinxcontrib-contentui](https://sphinxcontrib-contentui.readthedocs.io/en/latest/tabs.html) to use its tab-boxes anyway for this page, now that most of them have three tabs I think it's necessary. It's not a blocker for this PR, but you can add it now already if you like.

Reply:
I like the idea of tab boxes - that would also allow a proper grammar alongside examples. Let's make that a separate pull request, though.